packages/react-ui/components/Textarea/Textarea.tsx

```diff
@@ -290,6 +320,7 @@ export class Textarea extends React.Component<TextareaProps, TextareaState> {
       [jsStyles.textarea(this.theme)]: true,
       [jsStyles.error(this.theme)]: !!error,
       [jsStyles.warning(this.theme)]: !!warning,
+      [jsStyles.disableAnimation()]: this.isAnimationsDisabled(),
```

Comment:
Для консистентности, но не обязательно
```suggestion
      [jsStyles.disableAnimations()]: this.isAnimationsDisabled(),
```